src/amo/components/FlagReviewMenu/styles.scss

```diff
@@ -0,0 +1,5 @@
+@import '~amo/css/styles';
```

Comment:
Had to create a new am i missing something?>

Reply:
We do need a new file, but you shouldn't have to import `amo/css/styles` as this file isn't using anything from that file.